README.md

```diff
@@ -6,6 +6,21 @@ This project is intended to become a memory-safe implementation of NTP in Rust.
 
 We are currently looking for a better name for this project. Suggestions for this are welcome.
 
+## Binaries
+
+```
+cargo run
+```
+
+runs the NTP daemon
+
+```
+cargo run --bin demobilize
+```
+
+runs an NTP server on port `8080` that always responds with the DENY kiss code. It is used to test
```

Comment:
Nit: I would formulate this as it can be used instead of is used, since we don't use it in automated tests.